examples/with-wallet-stamper/src/components/turnkey-provider.tsx

```diff
@@ -139,7 +130,10 @@ export const TurnkeyProvider: React.FC<TurnkeyProviderProps> = ({
   }
 
   async function signInWithWallet(): Promise<User | null> {
-    const publicKey = (wallet as SolanaWalletInterface)?.recoverPublicKey();
+    const publicKey = await wallet?.getPublicKey();
```

Comment:
Reflects change to the wallet stamper interface, we now use `getPublicKey` for both EVM and Solana.